docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -548,201 +851,152 @@ func setGreeting(accessibleState PrecompileAccessibleState, caller common.Addres
 
-<!-- markdownlint-disable MD013 -->
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/feemanager"
 
-```go
-func (c *ChainConfig) GetActivePrecompiles(blockTimestamp *big.Int) PrecompileUpgrade {
-	pu := PrecompileUpgrade{}
-	if config := c.GetContractDeployerAllowListConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.ContractDeployerAllowListConfig = config
-	}
-	if config := c.GetContractNativeMinterConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.ContractNativeMinterConfig = config
-	}
-	if config := c.GetTxAllowListConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.TxAllowListConfig = config
-	}
-	if config := c.GetFeeConfigManagerConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.FeeManagerConfig = config
-	}
-  	if config := c.GetHelloWorldConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.HelloWorldConfig = config
-	}
-	// ADD YOUR PRECOMPILE HERE
-	// if config := c.{YourPrecompile}Config(blockTimestamp); config != nil && !config.Disable {
-	// 	pu.{YourPrecompile}Config = config
-	// }
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/rewardmanager"
 
-	return pu
-}
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/helloworld"
+	// ADD YOUR PRECOMPILE HERE
+	// _ "github.com/ava-labs/subnet-evm/precompile/contracts/yourprecompile"
+)
 ```
 
-<!-- markdownlint-enable MD013 -->
-
-Done! All we had to do was follow the comments.
-
-### Step 5: Add Precompile Upgrade
+### Step 6: Add Config Tests
 
-Let's add our precompile upgrade in `./params/config.go`. We can search (`CTRL F`) for
-`ADD YOUR PRECOMPILE HERE`. This file is used to set up blockchain settings.
+Let's add some unit tests to make sure our precompile is configured correctly. Config tests will
+be under `/precompile/contracts/helloworld/config_test.go`. There are mainly two functions we need
+to test: `Verify` and `Equal`. `Verify` checks if the precompile is configured correctly. `Equal`
+checks if the precompile is equal to another precompile. You can check other `config_test.go` files
+in the `/precompile/contracts` directory for examples. For the `HelloWorld` precompile, you can
+check the code in [here](https://github.com/ava-labs/subnet-evm/blob/helloworld-official-tutorial-v2/precompile/contracts/helloworld/config_test.go)
 
-Let's add the bool to check if our precompile is enabled. We are adding this to the `Rules` struct.
-`Rules` gives information about the blockchain, the version, and precompile status to functions that
-don't have this information. The below code snippet can be copied and pasted to overwrite the default
-code:
-
-```go
-  IsContractNativeMinterEnabled      bool
-  IsTxAllowListEnabled               bool
-  IsFeeConfigManagerEnabled          bool
-  IsHelloWorldEnabled                bool
-  // ADD YOUR PRECOMPILE HERE
-  // Is{YourPrecompile}Enabled       bool
-```
+### Step 6: Add Contract Tests
```

Comment:
There are two Step 6 and 7, please modify this and all steps below to make them unique.

Reply:
fixed all those steps